website/source/docs/connect/proxies/envoy.md

```diff
@@ -14,257 +14,309 @@ optionally exposing a gRPC service on the local agent that serves [Envoy's xDS
-              "portValue": 8080
-            }
-          },
-          "filterChains": [
-            {
-              "filters": [
-                {
-                  "name": "envoy.http_connection_manager",
-                  "config": {
-                    "stat_prefix": "public_listener",
-                    "route_config": {
-                      "name": "local_route",
-                      "virtual_hosts": [
-                        {
-                          "name": "backend",
-                          "domains": ["*"],
-                          "routes": [
-                            {
-                              "match": {
-                                "prefix": "/"
-                              },
-                              "route": {
-                                "cluster": "local_app"
-                              }
-                            }
-                          ]
-                        }
-                      ]
-                    },
-                    "http_filters": [
-                      {
-                        "name": "envoy.router",
-                        "config": {}
-                      }
-                    ]
-                  }
-                }
-              ]
-            }
-          ]
-        }
-        EOL
-    }
-  }
-}
-```
-
-#### Upstream Listener Configuration
-
-For the upstream listeners `proxy.upstreams[].config.envoy_listener_json`, no
-modification is performed. The `Clusters` served via the xDS API all have the
-correct client certificates and verification contexts configured so outbound
-traffic should be authenticated.
-
-Each upstream may separately choose to define a custom listener config. If
-multiple upstreams define them care must be taken to ensure they all listen on
-separate ports.
-
-Currently there is no way to disable a listener for an upstream, or modify how
-upstream service discovery clusters are delivered. Richer support for features
-like this is planned for the near future.
-
+specified via the bootstrap configuration and so Connect as of Consul 1.5.0 adds
+the ability to control some parts of the bootstrap config via proxy
+configuration options.
+
+The following configuration items may be added to the [global `proxy-defaults`
+configuration entry](#TODO) or overridden directly in the `proxy.config` field
+of a [proxy service
+definition](/docs/connect/proxies.html#proxy-service-definitions) or
+[`sidecar_service`](/docs/connect/proxies/sidecar-service.html) block.
+
+- `envoy_statsd_url` - A URL in the form `udp://ip:port` identifying a UDP
+  statsd listener that metrics should be delivered to. For example this may be
+  `udp://127.0.0.1:8125` if every host has a local statsd listener. In this
+  case it is convenient to configure this property once in the [global proxy
+  defaults](#TODO) config entry. Currently, TCP is not supported.
+
+    -> **Note:** currently the url **must use an ip address** not a dns name due
```

Comment:
```suggestion
    ~> **Note:** currently the url **must use an ip address** not a dns name due
```

Reply:
I think this should render as a yellow box since users might shoot themselves in the foot but it isn't a security risk?

Reply:
I Always forget out syntax and colour coding for there but yes a warning note makes sense.